utils/python/CIME/case_st_archive.py

```diff
@@ -178,7 +178,8 @@ def get_histfiles_for_restarts(case, archive, archive_entry, restfile):
         rundir = case.get_value("RUNDIR")
         cmd = "ncdump -v %s %s " %(rest_hist_varname, os.path.join(rundir, restfile))
         rc, out, error = run_cmd(cmd, ok_to_fail=True)
-        expect(rc == 0, "Command %s failed rc=%d\nout=%s\nerr=%s" %(cmd, rc, out, error))
+        if rc != 0:
```

Comment:
Yes. This variable is needed to tell you the variable name on the restart file that contains settings for what history files need to be in your exeroot on restart. What I did not catch earlier is that some components (like cam, clm) contain history restart files - and that variable will not be on those files. So at this point, I don't want to change the scheme of the the config_archive.xml  - so as a result we are just printing out a warning and continuing. I caught this in running the aux_clm45 tests with the latest cime.